packages/expo-localization/plugin/src/withExpoLocalization.ts

```diff
@@ -106,10 +117,11 @@ function withExpoLocalizationAndroid(config: ExpoConfig, data: ConfigPluginProps
     });
     config = withAppBuildGradle(config, (config) => {
       if (config.modResults.language === 'groovy') {
+        const bcp47Locales = supportedLocales.map((locale) => convertLocaleToBcp47(locale));
         config.modResults.contents = AndroidConfig.CodeMod.appendContentsInsideDeclarationBlock(
           config.modResults.contents,
           'defaultConfig',
-          `    resourceConfigurations += [${supportedLocales.map((lang) => `"${lang}"`).join(', ')}]\n    `
+          `    resourceConfigurations += [${bcp47Locales.map((lang) => `"${lang}"`).join(', ')}]\n    `
```

Comment:
Minor point on naming:the `resourceConfigurations` field is an array of resource qualifiers